tests/ui/code-location/expected

```diff
@@ -0,0 +1,7 @@
+module/mod.rs:10:5 in function module::not_empty
+main.rs:13:5 in function same_file
+~/.rustup/toolchains
```

Comment:
I'm not confident we should include this expectation, but this prevents us from inadvertently use `remapped` path. Open to suggestions.

Reply:
`RUSTUP_HOME` can be overriden (see https://rust-lang.github.io/rustup/environment-variables.html), so in the current form, the test may fail for users who override it.

If you're trying to make sure the path is not of the form: `/rustc/1eb72580d076935a3e590deb6e5813a5aef3eca4/library/core/src/mem/mod.rs`, perhaps just check for `toolchains/nightly-`?